tests/virt_autotest/login_console_as_kvm_or_xen_role.pm

```diff
@@ -0,0 +1,66 @@
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: For openSUSE virtualization test only. login in console, install kvm/xen patterns if needed.
+# Maintainer: Julie CAO <jcao@suse.com>
+
+use strict;
+use warnings;
+use base 'virt_autotest_base';
+use testapi;
+use ipmi_backend_utils;
+use Utils::Backends qw(use_ssh_serial_console is_remote_backend);
+use utils qw(zypper_call systemctl);
+use virt_autotest::utils qw(is_kvm_host is_xen_host);
+
+sub run {
+
+    #local openQA can skip TW host install
+    if (get_var('IPMI_DO_NOT_RESTART_HOST')) {
+        select_console 'sol', await_console => 0;
+        send_key 'ret';
+    }
+
+    use_ssh_serial_console !check_screen('text-logged-in-root');
+    return if get_var('IPMI_DO_NOT_RESTART_HOST');
+
+    zypper_call("--gpg-auto-import-keys ref",         180);
+    if (is_kvm_host) {
+        zypper_call("in -t pattern kvm_server kvm_tools", 900);     #for TW, tried 600 sec is not enough
```

Comment:
the comment is confusing. The default is 700s. Better not state times in comments as such comments can become wrong soon

Reply:
Good suggestion.